### DISCUSSION TITLE
Do we have the core power fx as npm package?

### DISCUSSION THREAD

**Discussion by psekaruipath** [microsoft/Power-Fx#94] [Q&A] [answered]
We are looking to consume power fx core language server and have our own editor on top in an angular application. As of now, I see that it is available only as a nuget package. Is it the case?

**Comment by MikeStall** [comment, marked as answer]
Check out the Formula bar sample: https://github.com/microsoft/Power-Fx/discussions/86
We do have a public NPM package for a react-based editor control.
The compiler itself is still C# and running on the server (and this is what's on the nuget feed), so the react control is making server calls for things like populating intellisense requests.  

You could also try running the PowerFx.core through blazor and running client side; but we haven't validated that yet.
